### PR TITLE
DEV-173: Re-enable shortcuts

### PR DESCRIPTION
**Pull request by PrestonB1123**
Re-added ctrl+n and ctrl+b as shortcuts with a default script.
https://highfidelity.atlassian.net/browse/DEV-173

### CODE REVIEW

File: scripts/system/keyboardShortcuts/keyboardShortcuts.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+//
+
+(function () { // BEGIN LOCAL_SCOPE
+    function keyPressEvent(event) {
+        if (event.text.toUpperCase() === "B" && event.isControl) {
+            console.log("TEST B");
+            Window.openWebBrowser();
+        }
+
+        else if (event.text.toUpperCase() === "N" && event.isControl) {
```
Comment (zfox23) [inline scripts/system/keyboardShortcuts/keyboardShortcuts.js]: remove the extra whitespace here please

File: interface/src/scripting/WindowScriptingInterface.h [hunk 2]
```diff
@@ -608,6 +608,12 @@ public slots:
      */
     void setActiveDisplayPlugin(int index);
 
+    /**jsdoc
+     * Opens `Browser.qml` in a pop-up window.
```
Comment (ZappoMan) [inline interface/src/scripting/WindowScriptingInterface.h]: let's change the description here to be a good user facing description. `Browser.qml` is an internal detail that users need not know about,

File: scripts/system/keyboardShortcuts/keyboardShortcuts.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+//  Copyright 2019 High Fidelity, Inc.
+//
+//  Distributed under the Apache License, Version 2.0.
+//  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+//
+
+(function () { // BEGIN LOCAL_SCOPE
+    function keyPressEvent(event) {
+        if (event.text.toUpperCase() === "B" && event.isControl) {
+            console.log("TEST B");
```
Comment (ZappoMan) [inline scripts/system/keyboardShortcuts/keyboardShortcuts.js]: log spam

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20189/HighFidelity-Beta-PR15856-1315677.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20189/HighFidelity-Beta-Interface-PR15856-1315677.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20189/HighFidelity-Beta-PR15856-1315677.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20189/HighFidelity-Beta-Interface-PR15856-1315677.exe

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20189/HighFidelity-Beta-PR15856-1315677.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20189/HighFidelity-Quest-Beta-PR15856-1315677.apk)

**Pull request merged**